src/lang/Messages.properties

```diff
@@ -1629,6 +1629,8 @@ report.unknown.error=Failed to write to file {0}, see log for detail
 report.write.overwrite.dialog.title=Existing File
 report.write.overwrite.dialog.message= The selected file already exists. Do you want to replace it?
 report.write.permission.dialog.title=Permissions Failure
+report.write.diskspace.warning.dialog.message=Warning: There may not be enough disk space to continue operation.
+report.write.diskspace.error.log.message=File creation may have failed or is not complete.
```

Comment:
The "diskspace" and "log" should be removed, the message applies to other errors and it's no longer just logged.